presto-main/etc/config.properties

```diff
@@ -41,3 +41,4 @@ plugin.bundles=\
 presto.version=testversion
 distributed-joins-enabled=true
 node-scheduler.include-coordinator=true
+experimental.iterative-optimizer-timeout=60s
```

Comment:
This config file is used for development (when running the presto server from your IDE). How does it relate to what you've described in the commit message?